### ISSUE TITLE
Unbelievably slow, trying to get customer ids and customer info

### ISSUE THREAD

**Issue by kingmeers** [Opteo/google-ads-api#503]
I believe this query is proving very slow

``` 
const customerListResponse = await googleAdsClient.listAccessibleCustomers(
      refreshToken
    );
```

And then, for each customer

```
const customerDetails = await customerService.query(`
            SELECT
              customer.id,
              customer.descriptive_name,
              customer.currency_code,
              customer.time_zone,
              customer.manager,
              customer.status
            FROM customer
`);
```
          
Is this expected since I chose so many fields? Or is there something wrong? For context, I am using the test api still, not basic or standard yet.

**Comment by visrut-at-handldigital** [comment]
Yes, it looks like the first query is slow, ended up switching to making calls using plain REST APIs instead and it's much faster.

v18: https://developers.google.com/google-ads/api/rest/reference/rest/v18/customers/listAccessibleCustomers

**Comment by kingmeers** [comment]
> Yes, it looks like the first query is slow, ended up switching to making calls using plain REST APIs instead and it's much faster.
> 
> v18: https://developers.google.com/google-ads/api/rest/reference/rest/v18/customers/listAccessibleCustomers

Thanks king will try this out 🙏

**Comment by smashah** [comment]
Why is it so slow compared to the REST API? Can anyone explain?
